docs/configuration.md

```diff
@@ -491,33 +491,30 @@ Use [globbing syntax](https://en.wikipedia.org/wiki/Glob_(programming)) for wild
 
 ## Baseline
 
-### `with-baseline` &lt;`flag`&gt; [`:committish`]
+### `baseline` &lt;`command`&gt;
 
-Default: `false`  
-Command line: `--with-baseline:feat-2`  
+Default: `false`
+Command line: `dotnet stryker baseline`
```

Comment:
dotnet stryker is not part of the command, this is required to run Stryker in the first place